fabric/fabric-commands/src/main/java/io/fabric8/commands/ProfileChangeParentsAction.java

```diff
@@ -60,9 +60,14 @@ protected Object doExecute() throws Exception {
         // we can only change parents to existing profiles
         Profile[] parents = FabricCommand.getExistingProfiles(fabricService, version, parentIds);
         ProfileBuilder builder = ProfileBuilder.Factory.createFrom(profile);
+        List<String> oldParents = builder.getParents();
         for (Profile parent : parents) {
             builder.addParent(parent.getId());
         }
+        //remove old parent profiles
+        for (String oldParent : oldParents) {
+            builder.removeParent(oldParent);
```

Comment:
@ffang what if someone wants to set a list of parents containing one of the older ones?